core/src/scss/utilities/variables/components/_nav.scss

```
@@ -0,0 +1,168 @@
+//  border--active:                  $color-vivid-red,
+//  link--expanded:                  $color-white,
+//  border--expanded:                transparent,
+//  border--active-expanded:         $color-white,
+//  h-link:                                   $color-black,
+//  h-link--hover:                            $color-bright-red,
+//  h-border--hover:                          $color-bright-red,
+//  h-link--active:                           $color-black,
+//  h-border--active:                         $color-black,
+//  h-link--active-hover:                     $color-bright-red,
+//  h-border--active-hover:                   $color-bright-red,
+//  h-link--expanded:                         $color-black,
+//  h-border--expanded:                       $color-black,
+//) !default;
+//
+// Styleguide: Variables.Components.nav-colors.default
+//
+
+$nav-colors: (
+    bg:                             $color-black,
+    divider:                        $color-cool-grey,
+
+    link:                           $color-white,
+    link--hover:                    $color-white,
+    link--active:                   $color-white,
+    link--expanded:                 $color-white,
+
+    border--hover:                  $color-white,
+    border--active:                 $color-vivid-red,
+    border--expanded:               transparent,
+    border--active-expanded:        $color-white,
+
+  // Horizontal menu colors (desktop Main Nav 1st level)
+    h-link:                                   $color-black,
+    h-link--hover:                            $color-bright-red,
+    h-link--active:                           $color-black,
+    h-link--active-hover:                     $color-bright-red,
+    h-link--expanded:                         $color-black,
+
+    h-border--hover:                          $color-bright-red,
+    h-border--active:                         $color-black,
+    h-border--active-hover:                   $color-bright-red,
+    h-border--expanded:                       $color-black,
+) !default;
+
+//
+// $nav-dark-colors
+//
+// Color map for the desktop version of the Main Nav dark variant 1st level items.
+//
+// Markup: $nav-dark-colors: (
+//  h-link:                               $color-white,
+//  h-link--hover:                        $color-white,
+//  h-border--hover:                      $color-white,
+//  h-link--active:                       $color-white,
+//  h-border--active:                     $color-vivid-red,
+//  h-link--active-hover:                 $color-white,
+//  h-border--active-hover:               $color-white,
+//  h-link--expanded:                     $color-white,
+//  h-link--expanded-hover:               $color-white,
+//  h-border--expanded:                   $color-white,
+//  h-border--expanded-hover:             $color-vivid-red,
+//  h-border--expanded-active:            $color-white,
+//) !default;
+//
+// Styleguide: Variables.Components.nav-colors.variant-dark
+//
+
+$nav-dark-colors: (
```

I would prefer a name more reminiscent of BEM naming:
```suggestion
$nav-colors--dark: (
```